test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/ExporterClientValidationTests.cs

```diff
@@ -0,0 +1,90 @@
+// <copyright file="ExporterClientValidationTests.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using OpenTelemetry.Exporter.OpenTelemetryProtocol.Implementation.ExportClient;
+using Xunit;
+
+namespace OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests
+{
+    public class ExporterClientValidationTests
+    {
+#if NETCOREAPP3_1
+        [Fact]
+        public void ExporterClientValidation_FlagIsEnabledForHttpEndpoint()
+        {
+            var initialFlagStatus = this.DetermineInitialFlagStatus();
+
+            try
+            {
+                var options = new OtlpExporterOptions
+                {
+                    Endpoint = new Uri("http://localhost:4173"),
```

Comment:
1. `http://localhost:4173` how about moving to a contant?
2. however, why it is even needed? it is the default value, right?

Reply:
To be explicit as the default value can change but I'll remove this if need be.

Reply:
I'm moving it to a const and keeping it since it's more explicit given we want to test against HTTP and HTTPS endpoints.